### ISSUE TITLE
Gwenview does not open AVIF or JXL anymore

### ISSUE THREAD

**Issue by KucharczykL** [NixOS/nixpkgs#351863]
### Describe the bug
Gwenview stopped opening AVIF or JXL formats. When I set up NixOS a few weeks back I believe this still worked as I used it quite often.

### Steps To Reproduce
1. Open AVIF or JXL.
2. See error

### Expected behavior
Gwenview supports AVIF and JXL.

### Additional context
I have `kdePackages.kimageformats` in `environment.systemPackages`.
When I open a JPG and press Ctrl+Shift+S to save it into a different format, WebP is available but AVIF nor JXL are in the list of supported formats.
In https://github.com/NixOS/nixpkgs/blob/master/pkgs/kde/frameworks/kimageformats/default.nix I see that `libjxl` and `libavif` are in `extraBuildInputs` but somehow that doesn't help?

### Notify maintainers

@ttuegel

### Metadata

---

- system: `"x86_64-linux"`
 - host os: `Linux 6.6.58, NixOS, 24.11 (Vicuna), 24.11.20241023.2768c7d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"home-manager-24.05.tar.gz, nixos-24.05"`
 - nixpkgs: `/nix/store/lsy6c2f9alj2gkjj36h754kk63x6701l-source`

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by FliegendeWurst** [comment]
#306227 may be relevant

**Comment by fin444** [comment]
Should be fixed by #349457 available on latest unstable.

**Comment by KucharczykL** [comment]
I can confirm that opening/saving AVIF/JXL in Gwenview and other programs like Spectacle is now fixed!

**Comment by KucharczykL** [comment]
I have to re-open this issue as Gwenview cannot open AVIF again - I am getting "Gwenview cannot display documents of type image/avif."

**Comment by fin444** [comment]
What is your version? I am on `25.11.20250704.5c724ed` and [this file](https://github.com/link-u/avif-sample-images/blob/master/fox.profile0.10bpc.yuv420.avif) opens fine in Gwenview.

**Comment by KucharczykL** [comment]
Seems like I'm on a newer version:
```
nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.39, NixOS, 25.11 (Xantusia), 25.11.20250719.c87b95e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.4`
 - channels(root): `"home-manager-24.05.tar.gz, nixos-24.05"`
 - nixpkgs: `/nix/store/182c2qj20sfspzxcshd6p051qdx3mx5j-source`
```

**Comment by fin444** [comment]
Yup, I can replicate on `25.11.20250723.fc02ee7`. Not sure why, since both `kimageformats` and `qtimageformats` are in the environment.

**Comment by fin444** [comment]
Nope, that's incorrect. `qimgv` [supports Qt6 as of 1.0.0](https://github.com/easymodo/qimgv/releases/tag/v1.0.0). The description in nixpkgs is out of date.

**Comment by fin444** [comment]
I just opened #482836 which now supports AVIF
